docs: stale PR response (#589)
electron/governance: playbooks/responses/stale-pr.md

---
action:
 - add 'stale' label
 - close pull request
---

Thank you for your contribution to the project!

> We apologize for the delay in reviewing your pull request. In the time since it

Reviewer: > We apologize for the delay in reviewing your pull request

I the case of most stale PRs, it isn't usually that there is a delay in reviewing, but rather a lack of response from the author of the PR.

Author: Good point. I've updated the wording and made a few other minor adjustments.

Thank you for your contribution to Electron!

> We've noticed there hasn't been any activity from the author in some time, and since the pull request was opened, there have been many changes to the codebase. As a result, your pull request may no longer be applicable or could conflict with more recent updates.

Reviewer 2: ```suggestion
We've noticed there hasn't been any activity on the pull request in some time, and there have been many changes to the codebase since it was opened. As a result, your pull request may no longer be applicable or could conflict with more recent updates.
```

I edited out the one instance of "the author" because it sounds weird when we're addressing them directly ("you") for the first of the paragraph.
